reana_workflow_controller/consumer.py

```diff
@@ -112,7 +112,6 @@ def _update_workflow_status(workflow, status, logs):
             WorkflowStatus.queued,
         ]
         if status not in alive_statuses:
-            workflow.run_finished_at = datetime.now()
```

Comment:
Question: Don't we have other occurrences of `workflow.run_finished_at/run_started_at/run_stopped_at` around? If I understood correctly these assignments will only happen when [the listener on `reana-db` is triggered](https://github.com/reanahub/reana-db/pull/87/files#diff-ce53bbb90bfc9cae2aa070829d3c6148R479), won't they?

Doing a search I could find [an occurrence of `run_started_at`](https://github.com/reanahub/reana-workflow-controller/blob/85376079f10e10111643ac515f5ee7079d21b657/reana_workflow_controller/rest/utils.py#L50), is this OK?

Reply:
Like always to the point 🎯 , thanks a lot for spotting:
```console
$ rg workflow.run_stopped_at
reana_workflow_controller/rest/utils.py
110:        workflow.run_stopped_at = datetime.now()
```
 I will amend :)

Reply:
Or maybe I'm wrong... 🤔  I see that specifically [`run_started_at` is set manually](https://github.com/reanahub/reana-workflow-controller/blob/85376079f10e10111643ac515f5ee7079d21b657/reana_workflow_controller/rest/utils.py#L50) while `run_stopped_at` and `run_finished_at` are set in the `reana-db` listener. So my question is, when [this status change happens](https://github.com/reanahub/reana-workflow-controller/blob/85376079f10e10111643ac515f5ee7079d21b657/reana_workflow_controller/rest/utils.py#L51), won't the `reana-db` listener be triggered so we can set `run_started_at` there instead of in `r-w-c`?